backend/backend/api/routes.py

```diff
@@ -130,12 +130,12 @@ async def read_root():
     </style>
```

Comment:
<<
Consider renaming the buttons from 'Test' and 'Tell Joke' to 'Research' and 'Get Joke', respectively, for clarity and consistency with the rest of the UI.
>>>